services/ui/ws/server_window.h

```diff
@@ -91,6 +91,12 @@ class ServerWindow {
                  const base::Optional<cc::LocalSurfaceId>& local_surface_id =
                      base::nullopt);
 
+  // Sets the bounds from manager. This is a reaction to resizes/moves done in
```

Comment:
SetBoundsFromHostWindowManager?

window manager as a meaning in mash context: it is ash itself. maybe make it explicit that we are talking about the host WM in the method name?